README.md

```diff
@@ -17,16 +17,33 @@ Tour of the source
 * `robotd/devices_base.py` contains some common code for `devices.py`.
 * `robotd.service` is the systemd service which runs the thing in production.
 
-Building the Debian package
+Getting started
+---------------
+
+Since `robotd` vendors in April Tags for its vision support, which depends on
```

Comment:
"vendors in" doesn't make much sense to me, and doesn't appear to be very commonly used in this way.
Does this mean that `robotd` includes April Tags when distributed? Or maybe that it depends on April Tags?

Reply:
@nmalky it did at the time, though the AprilTags stuff has since been moved out to https://github.com/sourcebots/sb-vision.

The idea of vendoring something into another project (perhaps not the best terminology) is to include some dependency (usually recursively, often in binary-form though not necessarily) in a given project. This is usually done for developer convenience and/or to pin a version. It's not really an ideal thing to do, but is useful as a way to get started.